Patch-Source: https://github.com/OmarCastro/rofi-blocks/pull/27
--
From 27c812142531ccb2dd8c22406ac85a8aa40fec94 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Tue, 1 Feb 2022 19:06:54 +0100
Subject: [PATCH] fix: Fix error unknown type name 'JsonNode' in page_data.h

Fixes the following compile error:

    In file included from check_page_data.c:4:
    ../src/page_data.h:59:56: error: unknown type name 'JsonNode'
       59 | void page_data_add_line_json_node(PageData * pageData, JsonNode * element);
---
 src/page_data.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/page_data.h b/src/page_data.h
index 5cd8f46..a4267d7 100644
--- a/src/page_data.h
+++ b/src/page_data.h
@@ -5,6 +5,7 @@
 #define ROFI_BLOCKS_PAGE_DATA_H
 #include <gmodule.h>
 #include <stdint.h>
+#include <json-glib/json-glib.h>
 typedef enum 
 {
 	MarkupStatus_UNDEFINED = 0,
